Face_contours/STEP/_STEP-face_surface_model_notes.txt

```diff
@@ -0,0 +1,9 @@
+2020-02-06
```

Comment:
Is this date correct?

Reply:
@darraghbr nope, you're 100% correct. I mistyped